### PR TITLE
change analytics to algorithm C

### PR DESCRIPTION
**Pull request by lbendavid-cloudinary**
#### What does this PR solve?
Implement algorithm C (OS type and version)

#### Final checklist
- [ ] Implementation is aligned to Spec.
- [ ] Tests - Add proper tests to the added code.

### CODE REVIEW

File: src/sdkAnalytics/getSDKAnalyticsSignature.ts [hunk 1]
```diff
@@ -31,6 +31,8 @@ function ensureShapeOfTrackedProperties(trackedAnalytics?: Partial<ITrackedPrope
     sdkCode: 'T', // Base code
     sdkSemver : packageVersion.split('-')[0], // remove -beta, -alpha or other tagged versions from the version string
     product: 'A',
+    osType: 'Z',
```
Comment (const-cloudinary) [inline src/sdkAnalytics/getSDKAnalyticsSignature.ts:34]: Don't we want to actually detect it?

**Review by const-cloudinary — CHANGES_REQUESTED** [review]
I find it useful to detect actual values for os type and version.

File: src/sdkAnalytics/getSDKAnalyticsSignature.ts (hunk 1 above)
Comment (adimiz1) [reply src/sdkAnalytics/getSDKAnalyticsSignature.ts:34]: I think it'll be dependent on the inheriting  framework
For example, React Native will be the one to detect the platform and the os version.

**Review by adimiz1 — APPROVED** [review]

File: src/sdkAnalytics/getSDKAnalyticsSignature.ts (hunk 1 above)
Comment (const-cloudinary) [reply src/sdkAnalytics/getSDKAnalyticsSignature.ts:34]: This package can be used as a standalone package in your pure JS project. We can detect os type and version.

**Pull request closed without merging**